RLBotFramework/gui/agent_frames/agent_frame.py

```diff
@@ -68,17 +67,17 @@ def refresh_widgets(self):
         for widget in self.grid_slaves(row=3, column=1):
             widget.grid_forget()
         if self.player_type.get() == "Human":
-            self.is_bot.set(False)
-            self.rlbot_controlled.set(False)
+            self.participant_type.set("human")
```

Comment:
this human and the Human above should be able to use the exact same value with a constant declared at the top of the file.

The same should go for the other bot types